lib/TimezoneGuesser/GuessFromOffset.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sabre\VObject\TimezoneGuesser;
+
+use DateTimeZone;
+use Sabre\VObject\Component\VTimeZone;
+
+class GuessFromOffset implements TimezoneGuesser
```

Comment:
What's this file?

Reply:
It's from the lib to find (GMT+offset) timezone

Reply:
Is this a rebase issue? I think we don't use this and now I'm also seeing changes to FindFromMzVersionTimezone?